privacyidea/lib/token.py

```diff
@@ -561,9 +561,9 @@ def get_one_token(*args, **kwargs):
     """
     result = get_tokens(*args, **kwargs)
     if not result:
-        raise ResourceNotFoundError(u"The requested token could not be found.")
+        raise ResourceNotFoundError(_(u"The requested token could not be found."))
```

Comment:
I think the 'u' is redundant, as the default string type is Unicode. In line 566 too

Reply:
after consultation it was decided to leave the 'u' standing.